src/runtime_src/tools/scripts/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ set(XRT_LOADER_SCRIPTS
   loader)
```

Comment:
Copyright date line 2 needs to be updated.

Reply:
yes, I will update this with next fix.